app/services/external_apis/ror_service.rb

```
@@ -158,53 +159,99 @@ def parse_results(json:)
       end
       # rubocop:enable Metrics/AbcSize
 
+      def ror_display_entry(item)
+        item['names'].find { |n| n['types'].include?('ror_display') }
+      end
+
+      # Extracts the org's display name from `names` to be used as sort_name
+      # "names": [
+      #     {"lang": "en", "types": ["ror_display","label"], "value": "Harvard University"},
+      #     {"lang": "es","types": ["label"], "value": "Universidad de Harvard"}
+      # ]
+      def sort_name(item)
+        ror_display_entry(item)&.fetch('value', nil)
```

We could just use `ror_display_entry(item)['value']` here.
Alternatively, `ror_display_entry(item).fetch('value')` should fail loud and catch the error you said was being missed here https://github.com/portagenetwork/roadmap/pull/1247/files#r2644246570

---

I'll go with `.fetch('value')`, a test revealed `['value']` would throw a nil error when there's no id or name so that's why I initially didn't go with it.